src/content/docs/browser/single-page-app-monitoring/get-started/browser-spa-v2.mdx

```diff
@@ -0,0 +1,91 @@
+* Focus on key metrics: While reported data remains largely unchanged, the new experience no longer tracks JavaScript execution or callback duration within interactions, streamlining the reported information.
+* Reduced interaction durations: Expect significantly shorter interaction durations, particularly for route changes. Initial page loads will see a slight reduction.
+* API updates:
+    * New optional argument `.interaction({waitForEnd: true})` added to the [`.interaction()`](https://docs.newrelic.com/docs/browser/new-relic-browser/browser-apis/interaction/) function: This allows customization of user interaction end time. The existing `.interaction()` functionality remains unchanged.
+    * Deprecated API: The function [`createTracer`](https://docs.newrelic.com/docs/browser/new-relic-browser/browser-apis/createtracer/): While still functional, the `createTracer` function is deprecated as it no longer keeps interactions open or times callbacks. Note: If you continue to use `createTracer` with the new SPA experience,`BrowserTiming` events will not be created.
+
+## Try out the new browser SPA monitoring experience [#enable-feature]
+
+<Steps>
+
+    <Step>
+        ### Review the requirements [#review-requirements]
+
+        This feature is tested and supported in line with our standard [browser support statement](https://docs.newrelic.com/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/#support-statement).
+
+
+    </Step>
+
+    <Step>
+        ### Add the feature flag [#add-feature-flag]
+
+        If your agent was installed with the APM method, contact our support team and we'll enable the feature flag on your account.
+        
+        If your agent was installed with NPM or the UI-based, copy/paste method, add the following feature flag your browser agent code:
+
+            1. Find the New Relic browser agent code in your webpage HTMl.
+
+            2. In the `init` configuration object, add the `soft_nav` feature flag. Here's an example:
+
+                ```js
+                <script type="text/javascript"> ;window.NREUM||(NREUM={});init={ …, feature_flags: ['soft_nav'] }:
+                ```
+
+            3. Deploy your app.
+
+        Need to disable this feature? Simply remove the feature flag.
+            
+    </Step>
+
+    <Step>
+        ### Confirm data is being sent to New Relic [#confirm-data]
+
+        First, check to make sure your interactions follow the heuristic: click/keydown/submit -> route change -> DOM modification.
+
+        Next, check the network tab in your browser’s dev tools. Filter for requests going to `/events/1/`. There should be ones for which the payload begins with `bel.7;1`, indicating your interactions are being captured and sent.
+
+        Finally, interaction data should continue to flow into your browser entity dashboards and charts.
+
+    </Step>
+
+</Steps>
+
+## Known issues [#known-issues]
```

Comment:
Should it be a part of release note? Or is it okay to place it here?

Reply:
It's okay to document known issues only for a limited/public preview where we're still working out the kinks.